packages/core/src/components/header/header.tsx

```diff
@@ -118,6 +118,7 @@ export class Header {
       return;
     }
 
+    this.setDropDownMenu();
```

Comment:
Dit gebeurd nu elke render. Mogelijk kom je uit de voeten met alleen `componentDidLoad()``

Reply:
Dat is inderdaad beter